WGS_pipeline/vcf_to_depmap.wdl

```diff
@@ -43,17 +43,13 @@ task vcf_to_depmap {
     }
 
     command {
-        pip install bioservices==1.10.1
-        pip install broad-genepy
-        git clone https://github.com/broadinstitute/depmap_omics.git
-        cd depmap_omics && git checkout dev && pip install -e . && cd ..
-        python depmap_omics/WGS_pipeline/vcf_to_depmap.py \
+        python /install/depmapomics/tasks/vcf_to_depmap.py \
               ~{input_vcf} \
               ~{sample_id} \
-              ~{n_rows} \
-              ~{use_multi} \
-              '~{sep="," force_keep}' \
-              ~{whitelist} \
+              --n_rows ~{n_rows} \
```

Comment:
Switched to using `--arg` names to be less sensitive to positional parameters. I think it's a good practice, that if something is optional *always* give it a name lest bugs get introduced when parameters are added or removed.